docs/Unicode.md

```diff
@@ -156,13 +154,13 @@ filename.ttf cannot be embedded due to copyright restrictions."
 
 # Fallback fonts #
 
-FPDF allows you to specify a list of fonts to be used if any character is not available on the font currently used. Common scenarios are use of special characters like emojis within your text, greek characters in formulas or citations mixing different languages.
+The method [`set_fallback_font()`](fpdf/fpdf.html#fpdf.fpdf.FPDF.set_fallback_font) allows you to specify a list of fonts to be used if any character is not available on the font currently set. When a character doesn’t exist on the current font, `fpdf2` will look if it’s available on the fallback fonts, on the same order the list was provided.
```

Comment:
typo alert: `set_fallback_fonts()` with a plural-s

Reply:
Well spotted, thank you!